docker/src/main/java/clocker/docker/location/DockerContainerLocation.java

```diff
@@ -188,12 +190,13 @@ public int execScript(Map<String,?> props, String summaryForLogging, List<String
         for (String commandString : filtered) {
             parseDockerCallback(commandString);
         }
-        if (getOwner().config().get(DockerContainer.DOCKER_USE_SSH)) {
+        boolean entitySsh = Boolean.TRUE.equals(entity.config().get(DockerContainer.DOCKER_USE_SSH));
+        boolean dockerSsh = Boolean.TRUE.equals(getOwner().config().get(DockerContainer.DOCKER_USE_SSH));
+        if (entitySsh && dockerSsh) {
```

Comment:
Why do we need it set on both the entity (i.e. the app component being deployed) and the owner (i.e. on the container)? What will set it on the container? Do you expect that to be explicitly set in the Clocker blueprint, or something more dynamic?